onnxruntime/core/framework/allocation_planner.cc

```diff
@@ -300,8 +300,8 @@ class PlannerImpl {
   const OrtValueNameIdxMap& ort_value_name_idx_map_;
 
   size_t num_logic_streams_{0};
-  std::vector<std::vector<NodeIndex>> stream_nodes_;
-  std::vector<size_t> node_stream_map_;
+  InlinedVector<InlinedVector<NodeIndex>> stream_nodes_;
```

Comment:
>nlinedVector<InlinedVector<NodeIndex>> [](http://example.com/codeflow?start=3&length=38)

This does not bring any benefit because even a single instance of InlinedVector can be stored inline.